### ISSUE TITLE
how to set up aws instances: instanceType and deviceMapping?

### ISSUE THREAD

**Issue by hongkailiu** [openshift/installer#671]
# Version

```console
$ ./bin/openshift-install version
./bin/openshift-install v0.3.0-208-g54b49cdf0fb6d1150c7a9d4102e2b785d5a0344d
Terraform v0.11.8

Your version of Terraform is out of date! The latest version
is 0.11.10. You can update by downloading from www.terraform.io/downloads.html
```

# Platform (aws|libvirt|openstack):

aws

# What happened?

I would like to know how to set up the instance type and device mapping
```
$ bin/openshift-install create manifests
$ vi ./tectonic/99_openshift-cluster-api_master-machines.yaml
...
        instanceType: m5.xlarge

$ bin/openshift-install create cluster --log-level=debug
```
1. the modification on `instanceType` did not have impact. The created instances are still with the default `t2.medium`
2. need a way to set up device mapping
```
$ oc version
oc v4.0.0-alpha.0+a148deb-572
kubernetes v1.11.0+d4cacc0
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://hongkliu-api.devcluster.openshift.com:6443
kubernetes v1.11.0+d4cacc0

$ oc explain machine
error: Couldn't find resource for "cluster.k8s.io/v1alpha1, Kind=Machine"
```

# What you expected to happen?

Would like to see how to set those properties up for aws instances.

# How to reproduce it (as minimally and precisely as possible)?

See above.

```console
$ your-commands-here
```

# Anything else we need to know?

This is more of questions instead of bug/issue report.
Would like to know how to do things with installer.

# References

- enter text here.

**Comment by abhinavdahiya** [comment]
So you can edit `install-config` be following https://github.com/openshift/installer/blob/master/docs/user/overview.md#multiple-invocations

Definition for options supported by installer:
https://godoc.org/github.com/openshift/installer/pkg/types#InstallConfig

> need a way to set up device mapping

I don't think installer supports that option.

**Comment by hongkailiu** [comment]
Thanks, @abhinavdahiya 
It feel that I did the right steps for changing the instanceType (from https://github.com/openshift/installer/blob/master/docs/user/overview.md#multiple-invocations it is not clear which file I should edit for this purpose):

```
$ bin/openshift-install create manifests
$ vi ./tectonic/99_openshift-cluster-api_master-machines.yaml
...
        instanceType: m5.xlarge

$ bin/openshift-install create cluster --log-level=debug
```

However, the resulting cluster is still with `t2.medium`.
Let me know what I missed. Thanks.

**Comment by wking** [comment]
#792 should fix this.

**Comment by akrzos** [comment]
Do we know when this might be merged so we can change instancetype of the initially deployed machines?

**Comment by wking** [comment]
Worker handling might be fixed by #1076 if someone wants to give that a spin.  #1088 is in flight with docs for adjusting these settings in the install-config.

**Comment by akrzos** [comment]
FWIW I can change the instancetype now:

Ex:
```
...
machines:
- name: master
  platform:
    aws:
      type: m5.large
  replicas: 3
- name: worker
  platform:
    aws:
      type: m5.large
  replicas: 3
...
```
